tdms/tests/unit/test_fields.cpp

```diff
@@ -0,0 +1,57 @@
+#include "catch2/catch_test_macros.hpp"
+#include "complex"
+#include "field.h"
+
+using namespace std;
+
+
+inline bool is_close(double a, double b){
+  return abs(a - b) < 1E-10;
```

Comment:
This definition is quite naive, it should at very least keep into account the scale of `a` and `b`.  `1e-100` is very much different from `1e-10` but with your definition they'd be equal.  Similarly, `1e100` and `1.0000000000000002e100` (the `double` number following `1e100`) are pretty close for any practical uses, but they'd be different according to this definition.  The docstring of Julia's [`isapprox`](https://docs.julialang.org/en/v1/base/math/#Base.isapprox) provides some details about how to better compare floating point numbers.

Reply:
fair enough – will modify accordingly. used |a - b| because it's easier to understand at a glance than |a - b|/max(a, b), when it's not hit